gef-ui/component-tests/partials/amendments/facility-value.component-test.ts

```diff
@@ -54,4 +55,40 @@ describe(page, () => {
 
     wrapper.expectText('[data-cy="facility-value-prefix"]').toRead(currencySymbol);
   });
+
+  it('should not render the error summary if there is no error', () => {
+    const wrapper = render(params);
+
+    wrapper.expectElement('[data-cy="error-summary"]').notToExist();
+  });
+
+  it('should not render the error summary if there is no error', () => {
```

Comment:
This test and the test above it have the same title - can you rename one to distinguish?